vividus-extension-selenium/src/main/java/org/vividus/converter/ui/AbstractParametersToSequenceActionConverter.java

```diff
@@ -14,50 +14,46 @@
  * limitations under the License.
  */
 
-package org.vividus.converter.ui.web;
+package org.vividus.converter.ui;
 
+import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
 import java.util.function.Supplier;
 
-import javax.inject.Named;
-
 import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
 import org.jbehave.core.steps.Parameters;
-import org.openqa.selenium.Point;
 import org.openqa.selenium.WebElement;
-import org.vividus.converter.PointConverter;
-import org.vividus.converter.ui.StringToLocatorConverter;
-import org.vividus.steps.ui.web.model.SequenceAction;
-import org.vividus.steps.ui.web.model.SequenceActionType;
+import org.vividus.steps.ui.model.SequenceAction;
+import org.vividus.steps.ui.model.SequenceActionType;
 
-@Named
-public class ParametersToSequenceActionConverter extends AbstractParameterConverter<Parameters, SequenceAction>
+public class AbstractParametersToSequenceActionConverter<T extends SequenceActionType<?>>
```

Comment:
Looks confusing Abstract... class that is not an abstract one, are there antPMD violations?

Reply:
Are there any subclasses for the converter?